### DISCUSSION TITLE
simple docker image + external nginx + lets encrypt

### DISCUSSION THREAD

**Discussion by RobertBerger** [Kovah/LinkAce#615] [Ideas & Feature Requests]
I run the simple docker image of linkace with http:// and everything works on my own network.
Now I would like to expose it via an nginx which runs on another server to the outside world.
The main advantage is, that I also have letsencrypt running there to update the keys and don't need to manually copy keys around.

Is there any way to do that?

Maybe external https:// access to the my nginx and this routes to the http:// docker image.

At the moment this fails after the login page ;)

How do other people update their keys automatically?

**Comment by RobertBerger** [comment]
probably I will need the advanced configuration?

**Reply by Kovah** [reply]
You can keep the simple version running and then use your existing server as a proxy. It will receive https requests and internally connects to your LinkAce.

See https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/ and https://www.linkace.org/docs/v1/setup/setup-with-docker/advanced-configuration/
